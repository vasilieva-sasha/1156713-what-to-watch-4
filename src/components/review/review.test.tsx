import * as React from "react";
import * as renderer from "react-test-renderer";
import Review from "./review";

const mock = {
  id: 0,
  user: {
    id: 0,
    name: ``,
  },
  comment: ``,
  date: `2020-07-09T16:06:01.831Z`,
  rating: 0,
};

describe(`ReviewComponent`, () => {
  it(`review component renders correctly`, () => {
    const tree = renderer.create(
        <Review review={mock} />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
