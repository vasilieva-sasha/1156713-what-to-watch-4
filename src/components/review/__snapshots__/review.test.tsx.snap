// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReviewComponent review component renders correctly 1`] = `
<div
  className="review"
>
  <blockquote
    className="review__quote"
  >
    <p
      className="review__text"
    >
      
    </p>
    <footer
      className="review__details"
    >
      <cite
        className="review__author"
      >
        
      </cite>
      <time
        className="review__date"
        dateTime="2020-07-09T16:06:01.831Z"
      >
        July 09, 2020
      </time>
    </footer>
  </blockquote>
  <div
    className="review__rating"
  >
    0
  </div>
</div>
`;
